lib/chef/resource/zypper_package.rb

```diff
@@ -30,6 +30,7 @@ class ZypperPackage < Chef::Resource::Package
 
       property :gpg_check, [ TrueClass, FalseClass ], default: lazy { Chef::Config[:zypper_check_gpg] }
       property :allow_downgrade, [ TrueClass, FalseClass ], default: false
+      property :global_options, [ String, Array ], coerce: proc { |x| x.is_a?(String) ? x.shellsplit : x }
```

Comment:
you should probably add a test which passed multiple options in a string and which then got shellsplit and that the call to `shell_out!()` wound up looking correct

Reply:
@lamont-granquist I've added unit test case for multiple options passed here https://github.com/chef/chef/pull/7518/files#diff-b638b95483df994a2982f5a0cfade9d3R175. Please take a look.